astroid/raw_building.py

```diff
@@ -179,13 +179,29 @@ def object_build_class(node, member, localname):
     return _base_class_object_build(node, member, basenames, localname=localname)
 
 
-def object_build_function(node, member, localname):
-    """create astroid for a living function object"""
-    signature = inspect.signature(member)
+def _get_args_info_from_callable(
+    member: Callable,
```

Comment:
Do we know more about this `Callable`?

Reply:
No, it can be any python callable.

Reply:
Can you more explicit and maybe suggest another typing for the function ?